packages/styleguide/src/templates/Article/base.js

```diff
@@ -35,20 +35,31 @@ import {
   matchImagesParagraph,
 } from './utils'
 import { MIN_GALLERY_IMG_WIDTH } from '../../components/Figure/Image'
+import { ExpandableLink } from '../../components/ExpandableLink'
+import { SEPARATOR as EXPANDABLE_LINK_SEPARATOR } from '../../components/ExpandableLink/ExpandableLink'
 
 const createBase = ({ metaBody, metaHeadlines }) => {
   const link = {
     matchMdast: matchType('link'),
-    props: (node) => ({
-      title: node.title,
-      href: node.url,
-    }),
+    props: (node, index, parent, { ancestors }) => {
+      const allowsExpandable = ancestors.find(matchParagraph)
```

Comment:
Unused and afaik always true anyhow (inline `link` always has a `paragraph` somewhere up the tree in mdast afaik)

Reply:
maybe I'm getting confused but i think that way you can't do expandable links in captions and such.

Reply:
if u check the variable: it only occurs 3 times and is never read for actual usage

Reply:
yeah ok, you're right…